test/unit/services/data-test.js

```diff
@@ -106,6 +107,18 @@ describe('InstallerDataService', function() {
     });
   });
 
+  describe('copyUninstaller', function(){
+    it('should be able to handle copyUninstaller', function(){
+      let data = {
+          darwin: ()=>'darwin',
+      };
+      sandbox.stub(Platform,'getOS').returns('darwin');
+      expect(Platform.identify(data)).to.be.equal('darwin');
+      sandbox.spy(svc, 'copyUninstaller');
```

Comment:
You need to call setup method here
``` JavaScript
svc.setup();
```
and this test will fail, because issue itself is not fixed yet.